django_airavata/static/common/js/layouts/ListLayout.vue

```diff
@@ -7,7 +7,16 @@
         </slot>
       </div>
       <div class="col-auto">
-        <slot name="additional-buttons"> </slot>
+        <slot name="additional-buttons"> 
```

Comment:
Instead of adding the buttons here, you can just use this slot in CredentialStoreDashboard.vue.  See ComputeResourceReservationList.vue for an example of adding an additional-buttons slot.